starter-code/memory.js

```diff
@@ -33,6 +33,16 @@ var MemoryGame = function() {
     this.correctPairs = 0;
 };
 
+MemoryGame.prototype.mixCards = function() {
```

Comment:
Whaat?? la funcion shuffle de lodash, si le pasas un array, te devuelve el mismo array desordenado. No hace falta crear variables extras. 
return this.cards=_.shuffle(this.cards)